string name
geometry_msgs/PoseStamped pose
